src/UpcomingBooking/UpcomingBooking.js

```diff
@@ -296,24 +301,34 @@ class UpcomingBooking extends React.Component<Props, State> {
     return (
       <div className="UpcomingBooking">
         <div className="Screen-title">
+          <div className="Screen-title-group">
+            <div className="booking-id">
```

Comment:
I believe there are separation margins of `4px` between the sibling div's `booking-id`, `title` and `${theStatusDiv}`.